From cd1579f0ed8f1c2fbfd7500fcd4f548cfe06e1f8 Mon Sep 17 00:00:00 2001
Message-ID: <cd1579f0ed8f1c2fbfd7500fcd4f548cfe06e1f8.1699195588.git.sharlatanus@gmail.com>
In-Reply-To: <1c3e20f81479ed702270fc6640939db165bd33bc.1699195588.git.sharlatanus@gmail.com>
References: <1c3e20f81479ed702270fc6640939db165bd33bc.1699195588.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 2 Nov 2023 15:16:35 +0000
Subject: [PATCH v3 2/5] gnu: qd: Update to 2.3.24.

During attempt to upgrade python-spherical-geometry there was upstream
discussion to adjust qd.  The patch was proposed and accepted by the
author of qd it was suggested by one one of spherical_geometry
maintainer:
https://github.com/spacetelescope/spherical_geometry/issues/227

* gnu/packages/multiprecision.scm (qd): Update to 2.3.24.
[native-inputs]: Add automake and autoconf.

Change-Id: Ic1dfdbe19b3347367b2ffb846be6bb975a0b89ae
---
 gnu/packages/multiprecision.scm | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/multiprecision.scm b/gnu/packages/multiprecision.scm
index 11afcfe4a0..0c6255c129 100644
--- a/gnu/packages/multiprecision.scm
+++ b/gnu/packages/multiprecision.scm
@@ -252,17 +252,16 @@ (define-public irram
 (define-public qd
   (package
     (name "qd")
-    (version "2.3.23")
+    (version "2.3.24")
     (source
      (origin
        (method url-fetch)
        (uri (string-append "https://crd-legacy.lbl.gov/~dhbailey/mpdist/qd-"
                            version ".tar.gz"))
        (sha256
-        (base32 "09pfd77rmy370hy7qdqw84z21y9zpl3fcwzf93rhiv0kwhfg9smk"))))
+        (base32 "0f0fwsyh23p5zfyrj6n16q7b56g213fnhdc8dbl22r3fz1rnqyx4"))))
     (build-system gnu-build-system)
-    (native-inputs
-     (list gfortran))
+    (native-inputs (list automake autoconf gfortran))
     (arguments
      (list
       #:configure-flags
-- 
2.41.0

